### ISSUE TITLE
Rake setup

### ISSUE THREAD

**Issue by jrich523** [tongueroo/cli-template#3]
Im on windows, so I've had all sorts of issues, but i believe this might be a non-os relatred issue.

Generated code looks like this
`cli.rb`

```
    def completion(*params)
      Completer.new(CLI, *params).run
    end
```

Which is in the module/class (top of the file)

```
module MyCli
  class CLI < Command
```
Rake tosses an error like this

```
cli.rb:25:in `completion': uninitialized constant MyCli::CLI::Completer (NameError)
Did you mean?  MyCli::Completer
               Complex
               MyCli::Completion

```
which, what it gives stands to make sense (to me, but im new)
Its looking locally in the CLI class to find the `Completer`, which it obviously wont find it

It still doesnt work if I change the call to look like MyCli::Completer (still learning class/module scope)

**Comment by tongueroo** [comment]
@jrich523 Wow a windows user, haven’t gotten too many questions from here 😄 Unsure when will be able take a look at this. 

Are you using RubyInstaller? Also what is the command you’re running when you get that error. See rake setup in the tittle. So rake?

**Comment by tongueroo** [comment]
@jrich523 Closing out until there's more info. Feel free to re-open.

**Comment by lucasprag** [comment]
@jrich523  I had the same problem, I found out that the `lib/your_module/completer.rb` was implementing the wrong module name, it was `Ufo`. I changed it to my module name and it worked.

**Comment by tongueroo** [comment]
@lucasprag Thanks! Didnt realize it'll be that quick to fix this. Reopening to take a look in time.

**Comment by lucasprag** [comment]
to reproduce that typo I mentioned:

- create a new CLI
```
$ cli-template new test
Creating new project called test.
      create  test
      create  test/test.gemspec
      create  test/.gitignore
      create  test/.rspec
      create  test/CHANGELOG.md
      create  test/Gemfile.lock
      create  test/Gemfile
      create  test/Guardfile
      create  test/LICENSE.txt
      create  test/README.md
      [..]
```

- check the `test/lib/test/completer.rb` file
```
cat test/lib/test/completer.rb
```

- you should see `Ufo` :
```
module Ufo # <- this is wrong
  class Completer
```


it looks like it's already fixed on [master](https://github.com/tongueroo/cli-template/commit/8735d2c0a884fc27a20741af7b44efc969e44e1a#diff-74d5919fea990a8c2afed5aece3d9e2aL71) but it was not published to rubygems.

FYI:
```
cli-template --version
3.3.0
```

**Comment by lucasprag** [comment]
I think you just need to bump the version and publish to master rubygems @tongueroo =)

**Comment by lucasprag** [comment]
Btw, nice job no this `gem`!

**Comment by tongueroo** [comment]
> RE: I think you just need to bump the version and publish to master rubygems @tongueroo =)

Sweet. Had a few other nice fixes in master also. Didnt even realize. Just released v3.4.0 [CHANGELOG](https://github.com/tongueroo/cli-template/blob/master/CHANGELOG.md)

> RE: Btw, nice job on this gem!

Thanks for the kind words!
